agent/config/parse_windows.go

```diff
@@ -114,3 +141,42 @@ var IsWindows2016 = func() (bool, error) {
 
 	return isWS2016, nil
 }
+
+var queryDomainlessGmsaPluginSubKeys = func() ([]string, error) {
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SYSTEM\CurrentControlSet\Control\CCG\COMClasses\`, registry.READ)
```

Comment:
We should make the registry keys as constants.

Reply:
Can I do this as a followup in a future cleanup PR? Sai is blocked on this work presently